ansible/files/gotrue-optimizations.service.j2

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=GoTrue (Auth) optimizations
+
+[Service]
+Type=oneshot
+# we don't want failures from this command to cause PG startup to fail
+ExecStart=/bin/bash -c "/opt/supabase-admin-api optimize auth --destination-config-file-path /etc/gotrue/gotrue.base.env ; exit 0"
```

Comment:
Let's follow existing convention and using "generated" in the file name rather than "base"

Reply:
Done.